src/FiskalyClient.ts

```diff
@@ -2,23 +2,33 @@ import * as jayson from 'jayson/promise';
 import { ClientConfiguration, VersionResponse, RequestResponse } from './responses';
 import { FiskalyErrorHandler, FiskalyError } from "./errors";
 import { ConfigureMethodParams, RequestMethodParams } from "./interfaces";
+import { ClientLibrary } from "./client/ClientLibrary";
 
 export class FiskalyClient {
     private readonly SDK_VERSION = '1.1.600';
     private context: string | undefined;
-    private readonly jsonRPC: jayson.Client;
+    private jsonRPC: jayson.Client | undefined;
+    private clientLibrary: ClientLibrary | undefined;
+    private hasService: boolean = true;
     private readonly allowedMethods = ['create-context', 'version', 'config', 'request', 'echo'];
 
     /**
      * Fiskaly Client Constructor.
-     * @param {string} fiskalyServiceUrl
+     * If fiskalyServiceUrl parameter not provided, the client library will be loaded from 'client' folder
+     * @param {string} fiskalyServiceUrl | optional
      */
     constructor(fiskalyServiceUrl: string) {
         if (!fiskalyServiceUrl) {
-            throw new FiskalyError("fiskalyServiceUrl must be provided");
+            try {
+                this.clientLibrary = new ClientLibrary();
+                this.hasService = false;
+            } catch (e) {
+                throw new FiskalyError("fiskalyServiceUrl must be provided");
+            }
+        } else {
+            // @ts-ignore
+            this.jsonRPC = jayson.Client.http(fiskalyServiceUrl);
```

Comment:
how about:
```suggestion
            try {
                const lib = new ClientLibrary();
                this.doRequest = lib.request.bind(lib);
            } catch (e) {
                throw new FiskalyError("fiskalyServiceUrl must be provided");
            }
        } else {
            const rpc = jayson.Client.http(fiskalyServiceUrl);
            this.doRequest = rpc.request.bind(rpc);
```

these class fields can also be removed then:
```ts
    private jsonRPC: jayson.Client | undefined;
    private clientLibrary: ClientLibrary | undefined;
    private hasService: boolean = true;
```

instead there would only be:
```ts
    private doRequest: Function;
```